docs/Basics.md

```diff
@@ -15,10 +15,34 @@ Running Exoframe server on its own also doesn't require too much resources:
 - Exoframe Server consumes ~50mb of RAM
 - Traefik started along with server consumes ~60mb of RAM
 
-Be aware though - execution of deployments will result in (1) new Docker images being built and (2) new Docker containers being started.  
+Be aware though - execution of deployments will result in:
+
+1. New Docker images being built. 
+2. New Docker containers being started.
+
 Depending on your project's complexity, this might require significant amount of resources during both steps resulting in failed deployments (note: if Docker goes out-of-memory during build, you will not get any specific error - just a failed deployment).  
 It is recommended to run Exoframe on a server with at least 1GB of RAM.
 
+## Deployment
+
+With Exoframe CLI installed, [create an `exoframe.json`](#Project-Config-File) file if not present and run
```

Comment:
This is inaccurate and needs to be rephrased.
1) You don't _need_ create `exoframe.json` if it's not there
2) It can be created by running `exoframe config` or `exoframe init`